docs/user_guide/migrating_versions/migrating_to_v0_3.rst

```diff
@@ -0,0 +1,32 @@
+Migrating to v0.3 from v0.2
```

Comment:
I think you need something like:
```.. _migrating_to_v0_3_0:```
here for the toctree link

Reply:
I'm so sorry I'm still not very familiar with the codes. Thanks for correcting me. I already added the code u mentioned.

Reply:
No worries, we're all here to learn. Thanks a lot for making these changes so quickly! The pipeline is running again :D